### ISSUE TITLE
Traning loss is negative

### ISSUE THREAD

**Issue by greecehalf** [kaushaltrivedi/fast-bert#187]
I am training a multilabel model using pretrained bert_uncased. The loss went down from 1 to and then continually went down to -1. I think it does not make sense. Does anyone have the same issue?
